independent-publisher-connectors/EncryptorPro/README.md

```diff
@@ -0,0 +1,22 @@
+### Encryptor Pro
+
+The Password Encryption Connector is a custom connector that provides secure encryption and decryption of sensitive data directly within your applications and workflows. This connector leverages embedded C# code to perform AES-256 encryption and decryption, ensuring robust data protection without the need for external services.
+
+## Prerequisites
+This connector does not require any API subscription or service to work.
+
+## Setting up the connector
+On initial use, you will need to initiate a new connection reference. This will allow for the connector to be used without any API keys or account IDs
+
+## Supported Operations
+The connector supports the following operations:
+* `Encrypt Data`:  Encrypts provided data into AES-256 encryption. 
+* `Dencrypt Data`: Decrypts encrypted data (encrypted using this connector) back into plain text.
+
+## Screenshots
+
+
+
+### For support see [support@turtledovecloudsolutions.com](mailto:support@turtledovecloudsolutions.cmo)
+
+### This connector is provided for use by the community to leverage the data ecnryption within a Microsoft Power Platform ecosystem.
```

Comment:
Could you please correct this typo "ecnryption"?

Reply:
Hi, this has been amended.